lib/App/index.js

```diff
@@ -2452,12 +2311,10 @@ class App {
       path.join('assets', 'images'),
     ];
 
-    // Append the homeycompose dir if used
-    if( answers.compose ) {
-      dirs.push( '.homeycompose' );
-      dirs.push( path.join('.homeycompose', 'flow') );
-      dirs.push( path.join('.homeycompose', 'drivers') );
-    }
+    // Append the homeycompose dir
+    dirs.push( '.homeycompose' );
+    dirs.push( path.join('.homeycompose', 'flow') );
+    dirs.push( path.join('.homeycompose', 'drivers') );
```

Comment:
Unrelated: should we add the other compose folders here as well? Such as `capabilities`, `signals`, `discovery` etc. Only `flow` and `drivers` feels a bit trivial to me.

Reply:
Sounds good!

Reply:
See: https://github.com/athombv/node-homey/pull/137